packages/style/scss/components/tooltip.scss

```diff
@@ -15,127 +12,37 @@
     }
 
     &.top,
-    &[data-popper-placement='top'] {
-        margin-top: -3px;
-        padding: 5px 0;
-
-        .tooltip-arrow {
-            bottom: 0;
-            left: 50%;
-            margin-left: -5px;
-            border-width: 5px 5px 0;
-
-            border-top-color: var(--digital-blue-100);
-        }
-    }
-
-    &.right,
-    &[data-popper-placement='right'] {
-        margin-left: 3px;
-        padding: 0 5px;
-
-        .tooltip-arrow {
-            top: 50%;
-            left: 0;
-            margin-top: -5px;
-            border-width: 5px 5px 5px 0;
-
-            border-right-color: var(--digital-blue-100);
-        }
-    }
-
     &.bottom,
+    &[data-popper-placement='top'],
     &[data-popper-placement='bottom'] {
-        margin-top: 3px;
-        padding: 5px 0;
-
-        .tooltip-arrow {
-            top: 0;
-            left: 50%;
-            margin-left: -5px;
-            border-width: 0 5px 5px;
-
-            border-bottom-color: var(--digital-blue-100);
-        }
+        margin: 12px 0;
```

Comment:
this is a tiny hack because the margin has no effect on absolute position 😅

Reply:
If it has no effect, how is using it here working? Not sure I understand what is the hack here :P

Reply:
Well it seems to take the label border as reference instead of the button border so  🤷‍♀️  feel like a hack xD